API.md

```diff
@@ -137,12 +137,18 @@ This function is not implemented yet.
 <div style="background-color: #E8E8E8; padding: 15px; margin: 10px; border-radius: 10px;">
 
 method CreateContainer() [NotImplemented](#NotImplemented)</div>
-This method has not been implemented yet.
+This method is not implemented yet.
 ### <a name="CreateFromContainer"></a>func CreateFromContainer
 <div style="background-color: #E8E8E8; padding: 15px; margin: 10px; border-radius: 10px;">
 
-method CreateFromContainer() [NotImplemented](#NotImplemented)</div>
-This method is not implemented.
+method CreateFromContainer(name: [string](https://godoc.org/builtin#string), image_name: [string](https://godoc.org/builtin#string), changes: [[]string](#[]string), author: [string](https://godoc.org/builtin#string), message: [string](https://godoc.org/builtin#string), pause: [bool](https://godoc.org/builtin#bool)) [string](https://godoc.org/builtin#string)</div>
+CreateImageFromContainer, also known as Commit, creates an image from an existing container. It requires the name or
```

Comment:
Is this CreateFromContainer or CreateImageFromContainer?

Reply:
its commit now